rpm build fails when python3 version is 3.4

When building RPMs for a RHEL 7.0 host (with python 3.4 RPMs built from Fedora 22 src rpms), I encountered the following build error.

```
git clone https://github.com/lxc/lxc.git
cd lxc
git checkout lxc-1.1.2 -b b_1_1_2
./configure
make rpm
```

```
Requires: /bin/bash /bin/sh /usr/bin/python3 libc.so.6()(64bit) libc.so.6(GLIBC_2.15)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libc.so.6(GLIBC_2.8)(64bit) libcap.so.2()(64bit) liblxc.so.1()(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) libselinux.so.1()(64bit) libutil.so.1()(64bit) rtld(GNU_HASH)
Processing files: lxc-libs-1.1.2-1.el7.x86_64
error: File not found by glob: /home/fpc/rpmbuild/BUILDROOT/lxc-1.1.2-1.el7.x86_64/usr/lib64/python3.3/site-packages/_lxc*
error: File not found by glob: /home/fpc/rpmbuild/BUILDROOT/lxc-1.1.2-1.el7.x86_64/usr/lib64/python3.3/site-packages/lxc/*


RPM build errors:
    File listed twice: /etc/sysconfig/lxc
    File not found by glob: /home/fpc/rpmbuild/BUILDROOT/lxc-1.1.2-1.el7.x86_64/usr/lib64/python3.3/site-packages/_lxc*
    File not found by glob: /home/fpc/rpmbuild/BUILDROOT/lxc-1.1.2-1.el7.x86_64/usr/lib64/python3.3/site-packages/lxc/*
make: *** [rpm] Error 1
```

The python3.3 version is hard-coded in the spec file. If changed to 3.4, the build works

```
%if %{with_python}
%{_libdir}/python3.4/site-packages/_lxc*
%{_libdir}/python3.4/site-packages/lxc/*
%endif
```

Would it be best to set the python version variable using autoconf?

--- a/lxc.spec.in
+++ b/lxc.spec.in
@@ -269,9 +269,6 @@ fi
 %{_sbindir}/*
 %{_libdir}/*.so.*
 %{_libdir}/%{name}
-%if %{with_python}
-%{_libdir}/python*
-%endif
 %{_localstatedir}/*
 %{_libexecdir}/%{name}
 %attr(4111,root,root) %{_libexecdir}/%{name}/lxc-user-nic
@@ -281,8 +278,7 @@ fi
 %endif
 
 %if %{with_python}
-%{_libdir}/python3.3/site-packages/_lxc*
-%{_libdir}/python3.3/site-packages/lxc/*
+%{python3_sitearch}/*
 %endif
 
 %if %{with_lua}

--- a/lxc.spec.in
+++ b/lxc.spec.in
@@ -252,13 +252,13 @@ fi
 %{_datadir}/doc/*
 %{_datadir}/lxc/*
 %{_sysconfdir}/bash_completion.d
-%{_sysconfdir}/sysconfig/*
 %config(noreplace) %{_sysconfdir}/lxc/*
 %config(noreplace) %{_sysconfdir}/sysconfig/*
 
 %if %{with_systemd}
 %{_unitdir}/lxc-net.service
 %{_unitdir}/lxc.service
+%{_unitdir}/lxc@.service
 %else
 %{_sysconfdir}/rc.d/init.d/lxc
 %{_sysconfdir}/rc.d/init.d/lxc-net
@@ -270,7 +270,9 @@ fi
 %{_libdir}/*.so.*
 %{_libdir}/%{name}
 %{_localstatedir}/*
-%{_libexecdir}/%{name}
+%{_libexecdir}/%{name}/hooks/unmount-namespace
+%{_libexecdir}/%{name}/lxc-apparmor-load
+%{_libexecdir}/%{name}/lxc-monitord
 %attr(4111,root,root) %{_libexecdir}/%{name}/lxc-user-nic
 %if %{with_systemd}
 %attr(555,root,root) %{_libexecdir}/%{name}/lxc-net

